src/app/components/mnoe-api/admin/sub-tenants.svc.coffee

```diff
@@ -0,0 +1,46 @@
+# Service for managing the users.
+@App.service 'MnoeSubTenants', (MnoeAdminApiSvc, MnoeObservables, OBS_KEYS) ->
+  @list = (limit, offset, sort) ->
+    MnoeAdminApiSvc.all("sub_tenants").getList({order_by: sort, limit: limit, offset: offset}).then(
+      (response) ->
+        response
+    )
+
+  @get = (id) ->
+    MnoeAdminApiSvc.one('sub_tenants', id).get()
+
+  @create = (subTenant) ->
+    promise = MnoeAdminApiSvc.all('sub_tenants').post(subTenant).then(
+      (response) ->
+        MnoeObservables.notifyObservers(OBS_KEYS.subTenantAdded, promise)
+        response
+      (error) ->
+        $log.error('Error while creating subTenant: ' + subTenant, error)
```

Comment:
It was my understanding that this was not a catch, but simply the second argument of then, taking a callback managing the error ?